content/docs/Other guides/customizing-rqd.md

```diff
@@ -43,79 +42,104 @@ To review the sample `Dockerfile`:
     ```bash
     cd ~/OpenCue
     ```
-
-1.  Run the following command to review the sample `Dockerfile`:
+1. Change to the subdirectory which includes the sample `Dockerfiles`.
 
     ```bash
-    cat samples/rqd/blender/Dockerfile
+    cd samples/rqd
     ```
 
-    The command outputs the contents of the `Dockerfile`.
+The sample `Dockerfiles` are listed within their respective subdirectories.
 
-    The first section of the file indicates that this `Dockerfile`
-    builds on the basic `opencue/rqd` container image hosted on
-    Docker Hub:
+### Reviewing sample Blender Dockerfile
```

Comment:
Add `the`. `Reviewing the sample Blender Dockerfile`

Same for the header below.

Reply:
Resolved in https://github.com/AcademySoftwareFoundation/opencue.io/pull/283/commits/272b44cc17a82a221da0025e3243f32dd5eec65b.

Will mark as _Ready for review_ once PR [#1327](https://github.com/AcademySoftwareFoundation/OpenCue/pull/1327) is merged.